docs/user/howto_config.md

```diff
@@ -262,6 +262,43 @@ sudo systemctl restart microshift
 oc get pods -n busybox
 ```
 
+### Deleting Manifests
+
+MicroShift also supports manifest deletion for removal or upgrade scenarios.
+Upgrade scenarios include situations where some objects should be removed, but not all of them to keep the data.
```

Comment:
May I suggest to reorder the paragraphs in this section?
* General statement
* Example of delete command and subdirs
* The "simple" delete scenario
* A more complex upgrade scenario with an example.